velox/exec/tests/HashJoinTest.cpp

```diff
@@ -2683,6 +2683,118 @@ TEST_P(MultiThreadedHashJoinTest, noSpillLevelLimit) {
       .run();
 }
 
+TEST_F(HashJoinTest, semiProject) {
```

Comment:
Curious, why was it necessary to move the test? It makes it hard to tell what's changed.

Reply:
Yeah, change from TEST_F to TEST_P to test with different number of drivers.